x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/inference/loadingservice/ModelLoadingServiceTests.java

```diff
@@ -451,6 +451,10 @@ public void testCircuitBreakerBreak() throws Exception {
         });
     }
 
+    public void testReferenceCounting() {
```

Comment:
think we need a test here or something? 

We might also want some tests to verify that our circuit breaker ends up back as zero. (Could be this same test).